projects/core/src/product/connectors/reviews/converters.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+import { Converter } from '../../../util/converter.service';
+import { Review } from '../../../occ/occ-models/occ.models';
+
+export const PRODUCT_REVIEWS_LIST_NORMALIZER = new InjectionToken<
```

Comment:
can we just name it `PRODUCT_REVIEWS_NORMALIZER`  and `PRODUCT_REVIEW_SERIALIZER`?

Reply:
ok